src/loaders/ObjectLoader.js

```diff
@@ -445,11 +448,25 @@ Object.assign( ObjectLoader.prototype, {
 
 	},
 
+	parseSkeletons: function ( json ) {
+
+		var skeletons = {};
+
+		for ( var i = 0; i < json.length; i ++ ) {
```

Comment:
Yeah. I am trying it out too.  This is a bug.  It definitely needs an undefined check here.